openmetadata-ui/src/main/resources/ui/src/components/common/OwnerLabel/OwnerLabel.interface.ts

```diff
@@ -29,4 +29,11 @@ export interface OwnerLabelProps {
   tooltipText?: string;
   isCompactView?: boolean;
   avatarSize?: number;
+  showMultipleType?: {
```

Comment:
showMultipleType seems a boolean from name and acting as object
Can you come up with something generic? 
 Like for ownerLabel it will be like `showEdit`? and caller may identify it based on different condition